packages/ipfs-core-types/src/root.d.ts

```diff
@@ -186,6 +186,11 @@ export interface AddOptions extends AbortOptions {
    */
   cidVersion?: CIDVersion
 
+  /**
+   * The multibase name to use. (defaults to 'base58btc' for v0 & 'base32' for v1)
+   */
+  multibaseName?: string
```

Comment:
Please use a `BaseName` type from [multibase](https://www.npmjs.com/package/multibase) instead of a string.